evervault_test.go

```diff
@@ -377,16 +308,10 @@ func TestClientInitClientErrorWithoutApiKey(t *testing.T) {
 	defer server.Close()
 
 	_, err := evervault.MakeClient("", "")
-
-	if err.Error() != evervault.ErrAppCredentialsRequired.Error() {
-		t.Errorf("Unexpected error, got error message %s", err)
-		return
-	}
+	assert.ErrorIs(t, err, evervault.ErrAppCredentialsRequired)
```

Comment:
Optional: the original test had a return after the check, so it makes sense to translate it into a require
```suggestion
	require.ErrorIs(t, err, evervault.ErrAppCredentialsRequired)
```